cmd/machine-config-daemon/main.go

```diff
@@ -14,8 +14,8 @@ const (
 var (
 	rootCmd = &cobra.Command{
 		Use:   componentName,
-		Short: "Run Machine Config Controller",
-		Long:  "",
+		Short: "Run Machine Config Daemon",
+		Long:  "Runs the Machine Config Daemon which handles communication between the host and the cluster",
```

Comment:
This should mentions that the daemon applies the machineconfigs to the machine. ?

Reply:
I can update it with that.

Reply:
Updated